### PR TITLE
WEBDEV-1557

### PR DESCRIPTION
**Pull request by kenzo7664**
Separate Astro Collections

### CODE REVIEW

File: src/components/Common/Footer.astro
Comment (louisjmorgan) [inline src/components/Common/Footer.astro:1]: I think you should use camelCase here like `projectsApplications` `projectsGames`,
or even just `applications` and `games` would probably be fine in this case

File: src/components/Projects/ProjectDetails.astro
Comment (louisjmorgan) [inline src/components/Projects/ProjectDetails.astro:1]: similarly, I would maybe rename the collections to either camelCase or kebab_case
kebab_case might work better for folder names, so applications_categories

File: src/content/config.ts
Comment (louisjmorgan) [inline src/content/config.ts:1]: use kebab_case here as well

File: src/layouts/CategoryLayout.astro
Comment (louisjmorgan) [inline src/layouts/CategoryLayout.astro:1]: kebab_case

File: src/layouts/ProjectLayout.astro
Comment (louisjmorgan) [inline src/layouts/ProjectLayout.astro:1]: kebab_case

**Review by louisjmorgan — CHANGES_REQUESTED** [review]
Looks good but you should use some kind of casing for variable names that combine multiple words. Please see the comments for details.

**Review by louismorgan-linaro — APPROVED** [review]
Looks good now.

**Comment by pcolmer** [comment]
As discussed, we are going to hold on merging these changes until we've had some data from Microsoft so that we can actually put some games in there rather than duplicating the application information.

**Comment by pcolmer** [comment]
I've converted this to a draft to prevent it from being merged.

**Comment by pcolmer** [comment]
Microsoft have replied now.

* Name is required
* Publisher is optional
* Compatibility status is required and must be one of Perfect, Playable, Runs, Unplayable
* Status description is optional
* Frame rate is optional and must be one of "less than 30 fps", "30-60 fps", "above 60 fps"
* Device configuration is optional
* OS Version is optional
* Date tested is required
* Game type is optional
* Additional notes is optional

Thanks.

**Comment by louismorgan-linaro** [comment]
Yes you're right. I'll just copy this https://github.com/sst/sst/blob/master/examples/quickstart-astro/.gitignore

**Comment by pcolmer** [comment]
As soon as the `.gitignore` file gets updated, I'm happy for this to be merged to staging.

**Pull request closed without merging**